src/components/NavBar/NavBar.js

```diff
@@ -4,9 +4,11 @@ import { Navbar, Nav, Button, Form } from "react-bootstrap";
 import logo from "../../images/logo.png";
 import Login from "../Login/Login";
 import "./NavBar.css";
+import SignUpModal from "../SignUpModal/SignUpModal";
 
 function NavBar() {
   const [showLogin, setShowLogin] = useState(false);
+  const [openModal, setOpenModal] = useState(false);
```

Comment:
still the state name needs to be better maybe smth that relates to signUpModal ?